### DISCUSSION TITLE
Add option to run part cooling fans during PID

### DISCUSSION THREAD

**Discussion by mrsav25** [mriscoc/Ender3V2S1#736] [Ideas]
Feature request: It seems to be debated whether part cooling fans should be on during PID tuning. I'm in the 'pro fans on' camp so I would like the option to set fans on (and ideally, select a % speed) for a PID tune (perhaps as a static setting somewhere else rather than part of the PID tuning UI flow).

IMO PID should match the printing conditions as much as possible, which includes having part cooling fans on.

UPDATE: Unless the part of the tune which comes up as "testing heat loss" accounts for this? It's a very short period but it looks like it puts the fans on for this reason.

**Comment by mriscoc** [comment]
You can set a Fan speed in Control / Temperature / Fan Speed

![temperature](https://user-images.githubusercontent.com/2745567/219127682-892f6f1e-3f8a-4132-a029-5fa2c420b9fe.jpg)

**Comment by mriscoc** [comment]
> "testing heat loss"

That is part of MPC not PID.

**Reply by mrsav25** [reply]
I thought MPC was the new built-in PID tuning on this firmware?

**Reply by mriscoc** [reply]
The latest versions have MPC instead of PID for the hot-end temperature control, but you can compile a custom version with PID if it is required.
